packages/@lwc/engine-core/src/framework/hooks.ts

```diff
@@ -97,11 +106,13 @@ enum LWCDOMMode {
 
 export function fallbackElmHook(elm: Element, vnode: VElement) {
```

Comment:
We might want to change the name of this function to something more representative. I was originally called this way to encapsulate the fallback logic when the LWC engine is not running with native shadow DOM.

Reply:
Hm, maybe `setStyleTokensAndObserve`? That seems to be what it's doing, for both synthetic shadow and light DOM scoped styles.

Or I could move the light DOM style scoping to a new hook.